src/js/consent-cookie.js

```diff
@@ -1,18 +1,35 @@
-const COOKIE_NAME = 'nhsuk-cookie-consent';
-const COOKIE_VERSION = 3;
-const COOKIE_LIVE_DAYS = 90;
+const COOKIE_NAME = 'cookiePreference';
+const COOKIE_DOMAIN = 'localhost'; // '.login.nhs.uk';
```

Comment:
How come we have a comment at the end of this? Also do we want to some logic to tell if we are on a nhs domain or localhost?

Reply:
Nope, it's just an oversight! Should always be '.login.nhs.uk'.